core/shared/src/main/scala/scalaz/zio/Managed.scala

```diff
@@ -0,0 +1,82 @@
+package scalaz.zio
+
+sealed abstract class Managed[+E, +R] { self =>
+  def use[E1 >: E, A](f: R => IO[E1, A]): IO[E1, A]
```

Comment:
Can we use `final` on all these methods?

Reply:
Yes, except for `use` :-)